pvanalytics/quality/irradiance.py

```diff
@@ -209,10 +227,11 @@ def check_irradiance_limits_qcrad(solar_zenith, dni_extra, ghi=None, dhi=None,
         Diffuse horizontal irradiance in :math:`W/m^2`
     dni : Series or None, default None
         Direct normal irradiance in :math:`W/m^2`
-    limits : dict, default QCRAD_LIMITS
-        for keys 'ghi_ub', 'dhi_ub', 'dni_ub', value is a dict with
-        keys {'mult', 'exp', 'min'} and float values. For keys
-        'ghi_lb', 'dhi_lb', 'dni_lb', value is a float.
+    limits : {'physical', 'extreme'} or dict, default 'physical'
+        If string, must be either 'physical' or 'extreme', corresponding to the
+        QCRAD QC limits. If dict, must have keys 'ghi_ub' and 'ghi_lb'. For
+        'ghi_ub' value is a dict with keys {'mult', 'exp', 'min'} and float
+        values. For 'ghi_lb' value is a float.
```

Comment:
```suggestion
        QCRAD QC limits. If dict, must have keys:
            
            * 'ghi_ub', 'dhi_ub', 'dni_ub': dicts with keys
              {'mult', 'exp', 'min'} and float values.
            * 'ghi_lb', 'dhi_lb', 'dni_lb': float values.
```

need to check that this renders as intended